Cargo.toml

```diff
@@ -46,7 +48,7 @@ rust-version = "1.66"
 
 [workspace.dependencies]
 # Dependencies maintained by sovereign
-jmt = "0.6.0"
+jmt = { git = "https://github.com/penumbra-zone/jmt" }
```

Comment:
Can we at least specify tag or commit, so it won't break if main branch there is broken